src/ralph/scan/plugins/ssh_cisco_catalyst.py

```diff
@@ -99,6 +99,8 @@ def get_subswitches(switch_version, hostname, ip_address):
 
 
 def scan_address(ip_address, **kwargs):
+    if 'juniper' in (kwargs.get('snmp_name', '') or '').lower():
```

Comment:
maybe `if 'juniper' in (kwargs.get('snmp_name') or '').lower():` - what if kwargs is dict with `snmp_name=None`? :)